### DISCUSSION TITLE
Darstellungsfilter Auswahlliste #

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#197] [Q&A]
Hallo,
beim Darstellungsfilter, kann man für eine Auswahlliste mit den ConnectionString  `#` direkt auf die Werte aus dem Service zugreifen. 

Darf da nur ein (1) Layer unter `Betroffene Layer` ausgewählt werden?

Oft sind Labelling und graphische Darstellung der Objekte in getrennten Layern. 
Wenn ich nun diese zwei Layer auswähle `Betroffene Layer` -  in beiden existiert die Spalte mit den Filterwerten, die durch `#` ausgelesen werden könnten.
Jedoch bekomme ich nur folgende  Fehlermeldung: 
`exception: "Can't determine required service layer object to get lookup values"`

LG
PS: 
![image](https://github.com/user-attachments/assets/e61f99be-92b6-4fcd-b348-b35e093e14ab)

**Comment by gis-eni** [comment]
Genau so ist es. Wenn du "komplexere" Datastellungsfilter baust, musst du wieder auf den konventionellen Weg zurück und die Auswahlliste über Datalinq oder eine Datenbank bereitstellen. Ein ``Distinct`` über mehrere Layer ist nicht möglich.

**Comment by marko-jer** [comment]
Hab ich befürchtet. Danke für die rasche  Antwort.
